homeassistant/components/pandora/media_player.py

```diff
@@ -173,13 +174,15 @@ def select_source(self, source: str) -> None:
             _LOGGER.warning("Station %s is not in list", source)
             return
         _LOGGER.debug("Setting station %s, %d", source, station_index)
+        assert self._pianobar is not None
```

Comment:
`self._pianobar` is only `None` if the media player entity is turned off. I think an `assert` makes the most sense in this case.